interface/LeptonEfficiencySF.h

```diff
@@ -863,22 +887,28 @@ class LeptonEfficiencySF
         }else if(id==169){
             if (is2016){
                 if(etaCen<0.9){
-                    if(etaFwd<0.9){ eff.first=0.9490; eff.second=0.001;
-                    }else if(etaFwd<1.2){ eff.first=0.9628; eff.second=0.005;
-                    }else if(etaFwd<2.1){ eff.first=0.9477; eff.second=0.005;
-                    }else if(etaFwd<2.4){ eff.first=0.9420; eff.second=0.010;
+                    if(etaFwd<0.9){ eff.first=0.9956; eff.second=0.0865;
```

Comment:
Hi @skeshri , as I told you the 0.0865 error should not be kept ! We know that it is coming from a bad behaviour of the fit and then it not real. I would fix it as it could introduce unexpected effect. Taking the number for the fit canvas (which is correct) I would replace it by 0.003